lib/rugged_adapter/git_layer_rugged.rb

```diff
@@ -659,6 +659,12 @@ def lstree(sha, options = {})
         results
       end
 
+      def lstree_all_directories(sha)
```

Comment:
Maybe `lstree_all_trees` to stay consistent with git terminology?

Reply:
That makes sense. I will change this.